README.md

```diff
@@ -13,7 +14,7 @@ Add it to your cargo.toml this will add the whole 2.0.x series as compatible as
 ```toml
 daemonize-me = "2.0"
 ```
-Then look at [example.rs](examples/example.rs)
+Then look at [hooks.rs](examples/example.rs)
```

Comment:
One small thing is updating the link here to the new filename

Reply:
Whoops!